receiver/signalfxreceiver/factory.go

```diff
@@ -108,7 +108,7 @@ func createMetricsReceiver(
 	}
 	receiverLock.Unlock()
 
-	r.RegisterMetricsConsumer(consumer)
+	r.registerMetricsConsumer(consumer)
```

Comment:
Similar question here. Or it's unrelated?

Reply:
Small nit here :D

Reply:
You would ask to move this to another PR haha

Reply:
Point taken, removed 🗡️